### PR TITLE
Make email attachments show as a graph edge

### PR DESCRIPTION
**Pull request by dwjohnston**
Solves #49 

I haven't tested this for this particular as I can't see an easy way for to run the code locally (I don't have python, jupiter notebook setup) #42 would help in this regard (provide a way to run/test the code locally, with JavaScript/Node only).

### CODE REVIEW

File: stix2viz/stix2viz/stix2viz.js [hunk 1]
```diff
@@ -585,6 +588,42 @@ define(["nbextensions/stix2viz/d3"], function(d3) {
             return;
           }
           Object.keys(item).forEach(function(key, index) {
+
+            // Iterate through the body_multipart array and do relationship linking there
+            if (key === 'body_multipart') {
```
Comment (dwjohnston) [inline stix2viz/stix2viz/stix2viz.js:593]: This is kind of quick and dirty very much a copy-paste kind of solution. But like, creating an abstraction might not be warranted.

**Comment by chisholm** [comment]
> I can't see an easy way for to run the code locally (I don't have python, jupiter notebook setup) #42 would help in this regard (provide a way to run/test the code locally, with JavaScript/Node only).

It is trivial to run locally.  Start your browser and open index.html directly from the filesystem.  E.g. press Ctrl+O, in the file browser navigate to index.html.  Using Node is actually more complicated than it needs to be.  This is hinted at in the readme:

> ... is suitable for standalone use — either on a hosted server or as a *local file* ...

but it's not really spelled out in detail.

As far as adding new embedded relationships, as Chris has [mentioned](https://github.com/oasis-open/cti-stix-visualization/issues/42#issuecomment-1082348385), we are in the process of changing graph libraries.  When that is settled, then we can resume building upon it.

**Comment by ejratl** [comment]
The desired behaviour is now present the visualizer following the merge of PR#52. I don't think this fix is needed any more. @dwjohnston will you check the latest visualizer and confirm whether it meets your needs, please?

**Comment by ejratl** [comment]
@dwjohnston I believe that the issue that you were trying to address with this pull request has been fixed. I am therefore closing this issue. Please reopen it if you disagree.

**Pull request closed without merging**